application/cms/form_fields.py

```diff
@@ -171,6 +181,7 @@ def __init__(self, label=None, validators=None, hint=None, **kwargs):
 
         super().__init__(label, validators, **kwargs)
         self.hint = hint
+        self.extended_hint = render_template(f"forms/extended_hints/{extended_hint}") if extended_hint else None
```

Comment:
I'm not sure if it's better to have the `render_template` here, or for us to call this at instantiation time - i.e. just pass in a block of HTML when we create the field in the form. 🤔

Reply:
Oh righto, yep, then this is good. Take a 👍